### ISSUE TITLE
connecting more than 1 iPhone/iPad fails

### ISSUE THREAD

**Issue by Grysu-Feuermelder** [dev47apps/droidcam#255]
**Describe the bug**
starting more than one instance of droidcam results in mapping all on the same v4l2loopback_dc-000.
Result is, that the output is flickering between both sources.  (Didn't check audio assignment, though)
This is the case for USB and WiFi (and on different ports)

**Expected behavior**
Registering more than one device correctly.

**Additional context**
Nice to have would be a feature to assign resolutions separately for each device.

System:
  Kernel: 5.15.0-82-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Cinnamon 5.8.4
    tk: GTK 3.24.33 wm: muffin dm: LightDM Distro: Linux Mint 21.2 Victoria base: Ubuntu 22.04 jammy

**Comment by aramg** [comment]
Multiple devices in parallel aren't supported (and not planned), though on Linux you can make it work by loading multiple v4l2loopback devices and then specifying a device to each instance of the client:
```
droidcam -dev=/dev/video2 -size=1280x720 ...
droidcam-cli -dev=/dev/video3 -size=1280x720 ...
```
